scss/typography/_helpers.scss

```diff
@@ -75,4 +98,16 @@ $stat-font-size: 2.5rem !default;
     margin-#{$global-left}: 0;
     list-style: none;
   }
+
+  .dropcaps {
+    line-height: $dropcaps-line-height;
+    &:first-letter {
+      margin-top: $dropcaps-first-letter-margin-top;
+      margin-bottom: $dropcaps-first-letter-margin-bottom;
+      padding-right: $dropcaps-first-letter-padding-right;
+      font-size: $dropcaps-first-letter-font-size;
+      line-height: $dropcaps-first-letter-line-height;
+      float: $dropcaps-first-letter-float;
+    }
```

Comment:
You can add: `text-transform: uppercase;` to prevent problematic letters like `q` or `p`.